x-pack/legacy/plugins/siem/public/pages/detection_engine/components/signals/index.tsx

```diff
@@ -59,13 +66,34 @@ export const SignalsTable = React.memo(() => {
     [setFilterGroup]
   );
 
+  const getUtilityBar = useCallback(
+    (totalCount: number) =>
+      filterGroup === FILTER_OPEN ? (
+        <OpenSignals totalCount={totalCount} />
+      ) : (
+        <ClosedSignals totalCount={totalCount} />
+      ),
+    [filterGroup]
+  );
+
+  const defaultIndices = useMemo(() => [`${DEFAULT_SIGNALS_INDEX}-default`], [
+    DEFAULT_SIGNALS_INDEX,
+  ]);
+  const defaultFiltersMemo = useMemo(
+    () => [
+      ...defaultFilters,
+      ...(filterGroup === FILTER_OPEN ? signalsOpenFilters : signalsClosedFilters),
+    ],
+    [defaultFilters, filterGroup]
+  );
```

Comment:
Status filter doesn't appear to be updating on filter change:
![signal_status_filter_not_working](https://user-images.githubusercontent.com/2946766/71025008-6ac18380-20c3-11ea-8fb9-4929fd740f7a.gif)

Also, the `defaultIndex` field on the gql request isn't being filled in. Might be a mis-match between `DEFAULT_SIGNALS_INDEX` and what's returned via `useFetchIndexPatterns()`??

<img src="https://user-images.githubusercontent.com/2946766/71025385-297da380-20c4-11ea-8f4c-d798e210d4f0.png" width=300 />